src/main/java/core/basesyntax/reportservices/ReportCreatorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.reportservices;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    @Override
+    public List<String> report(Map<String, Integer> transactions) {
+        List<String> report = new ArrayList<>();
+        report.add("fruit, quantity");
```

Comment:
The header 'fruit, quantity' is hardcoded. If the structure of the report changes, you will need to update this string. Consider defining this as a constant or providing it through a parameter.